src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -118,6 +124,30 @@ def _list_devices(self, cmd):
 
         return 0, result, ""
 
+    def _list_services(self, cmd):
+        svc_type = cmd.get('svc_type', None)
+
+        completion = self._oremote("list_services", svc_type)
+        self._wait([completion])
+        services = completion.result
+
+        if len(services) == 0:
+            return 0, "", "No services reported"
+        else:
+            # Sort the list for display
+            services.sort(key = lambda s: s.daemon_name)
+            services.sort(key = lambda s: s.nodename)
+            services.sort(key = lambda s: s.service_type)
+            lines = []
+            for s in services:
+                lines.append("{0}.{1} {2} {3}".format(
+                    s.service_type,
+                    s.daemon_name,
+                    s.nodename,
+                    s.container_id))
```

Comment:
Not all orchestrators can provide a proper `container_id`

Reply:
But they can put _something_ in there. Maybe we should relabel the field to something more generic -- "cluster_id" maybe? The rook backend can put the container_id in there -- others can put something else?